meta/open_standup.md

```diff
@@ -12,7 +12,7 @@ A standup is ideally ran by two people, one person talking, the other taking not
 * The person handling the talking parts `@here`s the dev channel. Include where the meeting is in the NYC office (usually the Classroom).
 * The same person reminds the project leads in #dev to prepare their tweet-sized updates, something like this:
 
-> friendly reminder: we’re doing tweet-sized updates from project leads during open standup so have yours ready! /cc @list @of @leads
+> friendly reminder: we’re doing summary updates from project leads during open standup, so have yours ready! /cc @kana @matt @alloy @acjay @chung-yi @joey
```

Comment:
I liked having a bit more of a quantifiable constraint and even then it proved hard to follow, so I’m a little weary of returning to possibly way too long summaries.

I think that this should be things that everybody can relate to, so high-level product efforts and large sweeping technical changes. I don’t want everybody to have to sit through minutes of updates on things that nobody outside a team can really understand.

Reply:
That’s all fair – I removed this because with fewer teams the updates didn’t need to be as condensed. Maybe instead of providing guidelines for length, we should provide guidelines for content? Like “here’s what your update should say” instead of “this is how long your update should be”?

Reply:
That makes sense to me 👍 Does my suggestion for content serve as a good basis?

Reply:
#29 did so